tests/snappi_tests/bgp/files/bgp_convergence_helper.py

```diff
@@ -662,7 +662,7 @@ def get_rib_in_convergence(snappi_api,
     route_names = NG_LIST
     bgp_config.events.cp_events.enable = True
     bgp_config.events.dp_events.enable = True
-    bgp_config.events.dp_events.rx_rate_threshold = 90/(multipath-1)
+    bgp_config.events.dp_events.rx_rate_threshold = 90/multipath
```

Comment:
why is the formula changing?

Reply:
We are advertising same routes from both the ports. Hence ECMP is formed .  Each Rx port will receive half of the Tx line rate. For ex if we are sending at 100% line rate, each port will receive 50% line rate. In order to latch DP Above timestamp, we have to set a value lesser than 50%. Earlier it was correct , in between in some commit this was changed due to which the convergence numbers were coming wrong.